suites/pacific/rados/tier-2_rados_test-pool-functionalities.yaml

```diff
@@ -198,10 +198,10 @@ tests:
   - test:
       name: Omap creations on objects
       module: test_omap_entries.py
-      polarion-id: CEPH-83571702
+      polarion-id: CEPH-83571702, CEPH-83572740
```

Comment:
Let's mark inactive one of the tests and have the steps merged into one.

Reply:
Removed the additional polarion id